packages/e2e-test-utils/README.md

```diff
@@ -183,7 +183,7 @@ _Parameters_
 
 _Returns_
 
--   `?puppeteer.ElementHandle`: Object that represents an in-page DOM element.
```

Comment:
I'd say it's probably already covered under #15186 or #17271, but this is an unfortunate loss of information as far as providing some context to what an ElementHandle actually is.

I would guess `docgen` should be enhanced to follow references of an `import`, linking somewhere relevant (either a first-party `@wordpress` package which documents the type, or a third-party package)

Reply:
Yes, I agree that it would be nice to have 👍